docs/notes_for_users.md

```diff
@@ -175,7 +173,56 @@ configurable options of _pydoclint_, and
 [here](https://jsh9.github.io/pydoclint/how_to_config.html) is how to configure
 _pydoclint_.
 
-## 6. How to adopt _pydoclint_ more easily in legacy projects
+## 6. Writing default arguments in numpy-style docstrings
+
+The [numpy docstring style guide](https://numpydoc.readthedocs.io/en/latest/format.html)
+specifies a few different ways to include default values for
+function args in the the docstring.
+
+For a function with a signature like this:
+
+```python
+def some_fn(arg1: int = 10):
+```
+
+The following styles are currently supported:
+
+```python
+    Parameters
+    ----------
+    arg1 : int, default 10
+
+    arg1 : int, default is 10
+
+    arg1 : int, default: 10
+
+    arg1 : int = 10
+```
+
+And for a function with a signature like this:
+
+``python
+def some_fn(arg1: int | None = None):
+
+````
+
+The following styles are currently supported:
+```python
+    Parameters
+    ----------
+    arg1 : int, optional
```

Comment:
Hi @mazer-ai , I think we probably need more work to specify the behavior here.

Because I think `int, optional` is a valid way in the docstring for `arg1: int | None = None` in the function signature, so we should add the logic to check this in the code.